proto/steward/v4/cellar_v2.proto

```diff
@@ -495,6 +568,21 @@ message CellarV2_5 {
             SetAlternativeAssetData set_alternative_asset_data = 17;
             // Represents function `setDropAlternativeAssetData(ERC20 _alternativeAsset)`
             DropAlternativeAssetData drop_alternative_asset_data = 18; 
+
+            // TEMPORARY
+            // These are governance functions, but will be allowed in a limited capacity for SPs while
+            // the new governance model is still in early adoption for emergencies.
+
+            // Represents function `setSharePriceOracle(uint256 _registryId, ERC4626SharePriceOracle _sharePriceOracle)`
+            SetSharePriceOracle set_share_price_oracle = 19;
+            // Represents function `cachePriceRouter(bool checkTotalAssets, uint16 allowableRange, address expectedPriceRouter)`
+            CachePriceRouter cache_price_router = 20;
+            // Represents function `forcePositionOut(uint32 index, uint32 positionId, bool inDebtArray)`
+            ForcePositionOut force_position_out = 21;
```

Comment:
This seems to be present in `3.x-main` but I think it's meant to be a gov function as per our conversations with Seven Seas.

Reply:
See also later comment, I see we are using it.